invenio_rdm_migrator/load/postgresql.py

```diff
@@ -144,7 +144,8 @@ def _post_load(self):
         """Post load processing."""
         tables = set()
         for tg in self.table_generators:
-            tables = tables.join(set(tg.tables))
+            tables = tables.union(set(tg.tables))
```

Comment:
question: why that change? what is the difference?

Reply:
join is not a defined function in a set 😓 

```
>>> set().union(set())
set()
>>> set().join(set())
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'set' object has no attribute 'join'
```